artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/etc/broker.xml

```diff
@@ -134,31 +134,36 @@ ${cluster-security.settings}${cluster.settings}${replicated.settings}${shared-st
             <expiry-address>ExpiryQueue</expiry-address>
             <redelivery-delay>0</redelivery-delay>
 
-            <!-- if max-size-bytes and max-size-messages were both enabled, the system will enter into paging
-                 based on the first attribute to hits the maximum value -->
-            <!-- limit for the address in bytes, -1 means unlimited -->
-            <max-size-bytes>-1</max-size-bytes>
+            <message-counter-history-day-limit>10</message-counter-history-day-limit>
+            <address-full-policy>${full-policy}</address-full-policy>
+            <auto-create-queues>${auto-create}</auto-create-queues>
+            <auto-create-addresses>${auto-create}</auto-create-addresses>
+            <auto-delete-queues>${auto-delete}</auto-delete-queues>
+            <auto-delete-addresses>${auto-delete}</auto-delete-addresses>
+
+            <!-- ************************************************************************************************************ -->
+            <!-- The size of each file                                                                                        -->
```

Comment:
I dont think all the seperators are needed, just the comment is fine. (Its also rather inconsistent with the rest of the file.)

```suggestion
            <!-- The size of each page file -->
```